cql-unit-tests/README.md

```diff
@@ -0,0 +1,25 @@
+## CQL Unit Tests
+
+The CQL Unit Tests are included in this repo as a companion JavaScript (TypeScript) module to the Covid-Ed-Ig CQL repository.
+These tests internally use the CQL-Exec-Fhir and CQL-Execution packages to run the CQL via the JavaScript engine.
+
+
+### Pre-requisites 
+Preferably install both a java version manager and a ruby version manager.
+Install the following:
+- Java 1.8
+- Ruby 3.0.0 
```

Comment:
I'm running Ruby 2.6.3 and was able to run the genonce script! So, you may not need 3.0.0

Reply:
Wooh! Does that mean you got Jekyll working?